DESCRIPTION

```diff
@@ -3,14 +3,18 @@ Package: bbr
 Title: R package for bbi
 Version: 1.2.1
 Authors@R: 
-    c(person(given = "Devin",
-             family = "Pastoor",
-             role = c("aut", "cre"),
-             email = "devin.pastoor@gmail.com"),
-      person(given = "Seth",
+    c(person(given = "Seth",
              family = "Green",
              role = "aut",
-             email = "seth127@gmail.com"),
+             email = "sethg@metrumrg.com"),
```

Comment:
You'll need to add "cre" to your roles, I think:

```
x System command 'R' failed, exit status: 1, stdout + stderr:
--
64 | E> * checking for file ‘/drone/src/DESCRIPTION’ ... OK
65 | E> * preparing ‘bbr’:
66 | E> * checking DESCRIPTION meta-information ... ERROR
67 | E> Authors@R field gives no person with maintainer role, valid email
68 | E> address and non-empty name.
69 | E>
70 | E> See section 'The DESCRIPTION file' in the 'Writing R Extensions'
71 | E> manual.
```

Reply:
yeaaa... I just noticed that. Just changed it and running `check()` locally to make sure nothing else weird happened. Thanks.